content/projects/tdd/simple-calculator-part1/_index.md

```diff
@@ -160,4 +160,14 @@ multiply(1,3)
 // should still return 3
 multiply(-1,3)
 // should still return -3
-```
\ No newline at end of file
+```
+
+## Instructions for reviewer
+
+* File structure is important, expected file structure displayed above.
+* `__init__.py` not needed if repo is set up properly (python).
+* The point of tests in TDD isn't only to have tests but to have very specific tests. For a given input we don't just want to know that they pass or fail. So instead of one function that tests everything you want specific tests.
+* Check for naming convention, use language's naming convention.
+* Do not use builtin function names as variable e.g. `sum`.
```

Comment:
Is this in the project instructions?

Reply:
no, but learners tend to do that which is bad practice, especially in this project